openrtb_ext/imp.go

```diff
@@ -4,29 +4,15 @@ import (
 	"encoding/json"
 )
 
-// ExtImp defines the contract for bidrequest.imp[i].ext
```

Comment:
We no longer have any references to `ExtImp` to extract `Prebid`? A little surprised, but I guess that is good.

Reply:
Correct. This looks like it was used before we moved to dynamic unmarshalling of the ext to find bidder info.